utils/src/unit-test/java/uk/gov/moj/sdt/utils/parser/XmlNamespaceUtilsTest.java

```diff
@@ -338,45 +339,43 @@ public void testMissingNamespace() {
 
-            Assert.fail("Failed to throw expected RuntimeException due to missing tag namespace.");
+            fail("Failed to throw expected RuntimeException due to missing tag namespace.");
         } catch (final RuntimeException e) {
-            Assert.assertEquals("Unrecognised exception message:", e.getMessage(),
-                    // CHECKSTYLE:OFF
-                    "Namespace [aop] missing from incoming raw xml[   <xsi:some-tag some-attribute=\"some value\">       <aop:some-other-tag some-attribute=\"some value\"/>       <aop:some-other-tag some-attribute=\"some value\">       </aop:some-other-tag>   </xsi:some-tag>]");
-            // CHECKSTYLE:ON
+            assertEquals("Namespace [aop] missing from incoming raw xml[   <xsi:some-tag some-attribute=\"some value\">       <aop:some-other-tag some-attribute=\"some value\"/>       <aop:some-other-tag some-attribute=\"some value\">       </aop:some-other-tag>   </xsi:some-tag>]",
+                    e.getMessage(), "Unrecognised exception message:");
         }
     }
 
     /**
      * Test the combining of individual namespaces.
      */
     @Test
-    public void testCombinedNamespace() {
-        final Map<String, String> map1 = new HashMap<String, String>();
-        final Map<String, String> map2 = new HashMap<String, String>();
+    void testCombinedNamespace() {
+        final Map<String, String> map1 = new HashMap<>();
+        final Map<String, String> map2 = new HashMap<>();
 
         // Setup individual maps.
         map1.put("xsi", "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"");
         map2.put("xsi", "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"");
         map2.put("aop", "xmlns:aop=\"http://www.springframework.org/schema/aop\"");
 
-        Map<String, String> combinedMap = new HashMap<String, String>();
+        Map<String, String> combinedMap = new HashMap<>();
         // Combine together the two maps.
         combinedMap = XmlNamespaceUtils.combineNamespaces(map1, combinedMap);
         combinedMap = XmlNamespaceUtils.combineNamespaces(map2, combinedMap);
 
-        Assert.assertEquals("Missing combined namespace", "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"",
-                combinedMap.get("xsi"));
-        Assert.assertEquals("Missing combined namespace", "xmlns:aop=\"http://www.springframework.org/schema/aop\"",
-                combinedMap.get("aop"));
-        Assert.assertEquals("Incorrect number of namespaces matching fragment", combinedMap.size(), 2);
+        assertEquals("xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"",
+                combinedMap.get("xsi"), "Missing combined namespace");
+        assertEquals("xmlns:aop=\"http://www.springframework.org/schema/aop\"",
+                combinedMap.get("aop"), "Missing combined namespace");
+        assertEquals(combinedMap.size(), 2, "Incorrect number of namespaces matching fragment");
```

Comment:
First two parameters of assertEquals() are the wrong way around.  Should be 2, combinedMap.size() - first is the expected value and second the actual value.